RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -39,7 +41,12 @@ public class SearchClusterContainer extends GenericContainer<SearchClusterContai
         "docker.elastic.co/elasticsearch/elasticsearch:5.6.16",
         Version.fromString("ES 5.6.16")
     );
-
+    public static final ContainerVersion ES_V2_4_6 = new OlderElasticsearchVersion(
+        "elasticsearch:2.4.6",
+        Version.fromString("ES 2.4.6"),
+        "/usr/share/elasticsearch/config/elasticsearch.yml",
+        "network.host: 0.0.0.0\npath.repo: \"/tmp/snapshots\""
```

Comment:
maybe worth a comment here about why the usual settings approach doesn't work and what's going on here.

Reply:
Mind if I merge and make this change on the follow up PR that I'll have out later today?

Reply:
Ended up making this change since I needed to resolve conflicts